src/labels/CustomLabels.labels

```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
+        <value>Learn more about Organization Donations in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationOrganizationPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Primary button label of checklist Item Donation Organization in GS for End User</shortDescription>
+        <value>Enter Organization Donations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationOrganizationTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Donation Organization in GS for End User</shortDescription>
+        <value>Enter a Donation from an Organization</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemTrackDonationDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Track Donation in GS for End User</shortDescription>
+        <value>Campaigns let you to track your constituent outreach in Salesforce. Use the Primary Campaign field on Opportunities to better evaluate your fundraising efforts.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemTrackDonationLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist Item Track Donation in GS for End User</shortDescription>
+        <value>Learn more about Campaigns in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemTrackDonationPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Primary button label of checklist Item Track Donation in GS for End User</shortDescription>
+        <value>Track Constituent Outreach</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemTrackDonationTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Track Donation in GS for End User</shortDescription>
+        <value>Track Donation Sources with Campaigns</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistTrackDonationDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Track Donation in GS for End User</shortDescription>
```

Comment:
I recommend changing this to: Description of Track Donation checklist in Get Started for End Users.